RadarSDK/RadarUtils.m

```diff
@@ -69,6 +69,35 @@ + (BOOL)locationBackgroundMode {
     return backgroundModes && [backgroundModes containsObject:@"location"];
 }
 
++ (NSString *)locationAuthorization {
+    CLAuthorizationStatus authorizationStatus = CLLocationManager.authorizationStatus;
+    switch (authorizationStatus) {
+        case kCLAuthorizationStatusAuthorizedWhenInUse:
+            return @"GRANTED_FOREGROUND";
+        case kCLAuthorizationStatusAuthorizedAlways:
+            return @"GRANTED_BACKGROUND";
+        default:
+            return @"DENIED";
+    }
+}
+
++ (NSString *)locationAccuracyAuthorization {
+    CLLocationManager *locationManager = [CLLocationManager new];
+    if (@available(iOS 14.0, *)) {
+        CLAccuracyAuthorization accuracyAuthorization = locationManager.accuracyAuthorization;
+        switch (accuracyAuthorization) {
+            case CLAccuracyAuthorizationReducedAccuracy:
+                return @"REDUCED";
+            case CLAccuracyAuthorizationFullAccuracy:
+                return @"FULL";
+            default:
```

Comment:
Will this default ever be reached?

Reply:
It shouldn't, at least on the current version of iOS (not sure if Apple will add more enum values in the future)